tests/audit-scanner-installation.bats

```diff
@@ -15,8 +15,8 @@ setup() {
     kubectl create -f https://github.com/kubernetes-sigs/wg-policy-prototypes/raw/master/policy-report/crd/v1alpha2/wgpolicyk8s.io_policyreports.yaml
     kubectl create -f https://github.com/kubernetes-sigs/wg-policy-prototypes/raw/master/policy-report/crd/v1alpha2/wgpolicyk8s.io_clusterpolicyreports.yaml
 
-    helm_in kubewarden-crds  --set installPolicyReportCRDs=False
-    helm_in kubewarden-controller  --set auditScanner.enable=True
+    helm_in kubewarden-crds --set installPolicyReportCRDs=False
+    helm_in kubewarden-controller
     helm_in kubewarden-defaults  \
         --set recommendedPolicies.enabled=True \
         --set recommendedPolicies.defaultPolicyMode=protect \
```

Comment:
@jvanz can you also pls confirm that we always want to use latest policyServer image?

Reply:
I think we should always use the policyServer image that is defined on the `kubewarden-defaults` chart.

In the future, we could run the e2e tests in the github.com/kubewarden/policy-server repo, and there swap the image for one built from the PR or main.